doc/nrf/drivers/wifi/nrf70_native.rst

```diff
@@ -0,0 +1,120 @@
+
+The driver supports two modes of operation:
+
+Wi-Fi mode
+==========
+
+In this mode, the driver is designed to be used with the Zephyr networking stack.
+It is implemented as a network interface driver.
+
+The driver supports the following IEEE 802.11 features:
+
+* Wi-Fi 6 (802.11ax) support
+* WPA3™/WPA2™ personal security
+* IEEE 802.11 Power Save modes
+* Scan-only mode
+* IEEE 802.11 :term:`Station mode (STA)`
+* :term:`Software-enabled Access Point (SoftAP or SAP)` mode
+
+The Wi-Fi Direct® mode feature is in the driver code but is not yet supported.
+
+Except for scan-only mode, the driver uses the host access point daemon (hostapd) to implement AP Media Access Control (MAC) Sublayer Management Entity (AP MLME) and wpa_supplicant to implement 802.1X supplicant.
+
+Radio Test mode
+===============
+
+The nRF Wi-Fi driver supports Radio Test mode, which you can use to test the RF performance of the nRF70 Series device.
+This is a build time option that you can enable using the :kconfig:option:`CONFIG_NRF70_RADIO_TEST` Kconfig option.
+
+For more details about using this driver in Radio Test mode, see :ref:`wifi_radio_test`.
+
+Driver to nRF70 Series device communication
+*******************************************
+
+The driver communicates with the nRF70 Series device using the QSPI/SPI interface.
+The driver uses the QSPI/SPI interface to send commands to the nRF70 Series device, and to transfer data to and from the device.
+The nRF7002 DK uses QSPI, whereas the nRF7002 EK uses SPI.
+
+To connect the nRF7002 EK to the SoC, the ``nrf7002ek`` shield is required.
+
+Configuration
+*************
+
+The nRF Wi-Fi driver has the following configuration options:
+
+Kconfig configuration
+=====================
+
+.. options-from-kconfig:: /../../../../../zephyr/drivers/wifi/nrfwifi/Kconfig.nrfwifi
+   :show-type:
+
+Devicetree specification configuration
+======================================
+
+The maximum transmit power achieved on an nRF70 Series device-based product depends on the frequency band and operating channel.
+This varies across different :term:`Printed Circuit Board (PCB)` designs.
+
+Multiple calibrations and checks are implemented to ensure consistency across channels and devices.
+However, these values depend on PCB design, which may result in Error Vector Magnitude (EVM) and spectral mask failures.
+To avoid this problem, you can specify the power ceiling at which the EVM and spectral mask are met for a given PCB design.
+Additionally, build-time parameters are made available to drivers through the DTS overlay file.
+
+The following code snippet shows an example of the DTS overlay file.
+Note that the numbers used in this following example do not represent any particular PCB design or package type.
+
+You must replace these values with measurements obtained from transmitter testing on your own PCB designs.
+The values are represented in 0.25 dB increments.
```

Comment:
This is not true, during upstreaming based on review comments, we have made this generic, so, removed the custom step size and used 1dBM, so, this and example values should be scaled down. Look at https://github.com/nrfconnect/sdk-zephyr/blob/main/drivers/wifi/nrfwifi/src/fmac_main.c#L497